test/NuGet.Tests.Apex/NuGet.Console.TestContract/ApexTestConsole.cs

```diff
@@ -21,17 +22,22 @@ public ApexTestConsole(IWpfConsole WpfConsole)
             _wpfConsole = WpfConsole;
         }
 
-        private void EnsureInitilizeConsole()
+        private bool EnsureInitilizeConsole()
         {
-            while (!_wpfConsole.Dispatcher.IsStartCompleted || _wpfConsole.Host == null)
+            _wpfConsole.Dispatcher.Start();
+            var stopwatch = Stopwatch.StartNew();
+            var timeout = TimeSpan.FromSeconds(5);
```

Comment:
5 seconds seems fast for what could be a very slow test VM. I would be fine with this taking up to 5 minutes as long as it means I don't get a failure for no reason. Since this checks every 100ms I think a long timeout is fine.

Reply:
@emgarten should I also update the timeout when running command (line 43 of this same file) to be 5 minutes instead of 5 seconds?